client/src/lib/components/TagSelector/TagSelector.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="ts">
+  import type { SelectOption } from "src/definitions/form";
+  import type { Tag as TagType } from "src/definitions/tag";
+  import { transformTagToSelectOption } from "src/lib/transformers/form";
+  import { createEventDispatcher } from "svelte";
+  import Search from "../Search/Search.svelte";
+  import Tag from "../Tag/Tag.svelte";
+
+  export let tags: TagType[];
+  export let name: string;
+  export let id: string = name;
+
+  export let selectedTags: TagType[] = [];
+
+  const dispatch = createEventDispatcher();
```

Comment:
L'avantage de faire ça dès qu'on le peut c'est que la cohérence de typage des handlers est aussi vérifiée dans les composants parents (si un handler prend autre chose qu'un `CustomEvent<Tag[]>` TS renverra une erreur).

```suggestion
  const dispatch = createEventDispatcher<{ change: TagType[] }>();
```